src/test/groovy/no/skatteetaten/aurora/boober/service/AuroraDeploymentSpecValidatorTest.groovy

```diff
@@ -12,6 +12,16 @@ class AuroraDeploymentSpecValidatorTest extends AbstractAuroraDeploymentSpecTest
 
     def specValidator = new AuroraDeploymentSpecValidator(openShiftClient)
 
+    def "Fails when envFile does not start with about"() {
+      given:
+        auroraConfigJson["utv/aos-simple.json"] = '''{ "envFiles": "foo" }'''
+
+      when:
+        createDeploymentSpec(auroraConfigJson, DEFAULT_AID)
```

Comment:
This test is actually testing at an inappropriate level of abstraction for this particular test class. This class should only test the AuroraDeploymentSpecValidator (which verifies that the AuroraDeploymentSpec is valid - particularly with regards to object references like templates and AD groups).

What your test is doing is testing that creating an AuroraDeploymentSpec fails when the AuroraConfig is invalid - which is actually at a different layer of abstraction.

We should probably move most of the AbstractAuroraDeploymentSpecTest into a new class (AbstractAuroraConfigTest) that we put in the model package. And then create a new test class in the model package for testing AuroraDeploymentSpecBuilder.createAuroraDeploymentSpec. This test class could be used to verify that all sorts of error conditions in all sorts of AuroraConfig setups are handled correctly.